Student.cpp

```diff
@@ -0,0 +1,126 @@
+#include "Student.hpp"
+
+Student::Student(const string& vardas, const string& pavarde) {
+    this->vardas = vardas;
+    this->pavarde = pavarde;
+}
+
+Student::~Student() = default; // destruktorius
+
+Student::Student(const Student & student) { // kopijavimo konstruktorius
+    vardas = student.vardas;
+    pavarde = student.pavarde;
+    pazymiai = student.pazymiai;
+    egzamino = student.egzamino;
+    pazymysGalutinis = student.pazymysGalutinis;
+    pazymiuVid = student.pazymiuVid;
+    pazymiuMed = student.pazymiuMed;
+}
+
+Student& Student::operator=(const Student& student) { // kopijavimo priskyrimo operatorius
+    if (this == &student) {
+        return *this;
+    }
+
+    vardas = student.vardas;

```

Comment:
You could reuse Student:Student copy constructor. This will cleanup the code a little.